src/firefly/js/visualize/ui/VisCtxToolbarView.jsx

```diff
@@ -94,65 +86,104 @@ function changeAutoConvert(pv, auto) {
+                {isHiPS(plot) && <DropDownVerticalSeparator useLine={true}/>}
+            </>}
+            {isHiPS(plot) &&
+                <>
+                    <Typography>HiPS Projection</Typography>
+                    <ToolbarButton {...{
+                        hasCheckBox: true,
+                        checkBoxOn: !isHiPSAitoff(plot),
+                        key: 'change Hips',
+                        text: SPHER_TEXT,
+                        tip: 'All-sky multi-resolution picture with spherical projection, up to 180 degrees',
+                        onClick: () => dispatchChangeCenterOfProjection({plotId: pv.plotId, fullSky: false})
+                    }}/>
+                    <ToolbarButton {...{
+                        hasCheckBox: true,
+                        checkBoxOn: isHiPSAitoff(plot),
+                        key: 'change aitoff',
+                        text: AITOFF_TEXT,
+                        tip: 'All-sky multi-resolution picture with AITOFF projection, up to 360 degrees',
+                        onClick: () => dispatchChangeCenterOfProjection({plotId: pv.plotId, fullSky: true})
+                    }}/>
+                    <DropDownVerticalSeparator useLine={true}/>
+                    <Typography sx={{whiteSpace:'nowrap'}}>HiPS Coordinate System</Typography>
+                    <ToolbarButton text='Galactic' tip='Use Galactic coordinate system' key={'gal'}
+                                   hasCheckBox={true}
+                                   checkBoxOn={plot.imageCoordSys===CoordinateSys.GALACTIC}
+                                   onClick={()=>dispatchChangeHiPS( {plotId,  coordSys: CoordinateSys.GALACTIC})}/>
+                    <ToolbarButton text='Eq J2000' tip='Use Equitorial J2000 coordinate system' key={'eqj'}
+                                   hasCheckBox={true}
+                                   checkBoxOn={plot.imageCoordSys===CoordinateSys.EQ_J2000}
+                                   onClick={()=>dispatchChangeHiPS( {plotId,  coordSys: CoordinateSys.EQ_J2000})}/>
+                </>
+            }
+        </SingleColumnMenu>
     );
-}
-
-HipsFitsConvertButton.propTypes= { pv : object.isRequired};
-
 
 
-function HipsProjConvertButton({pv}) {
-
-    const plot= primePlot(pv);
-    if (!plot) return undefined;
-    const value= isHiPSAitoff(plot) ? 'aitoff' : 'sin';
-    const buttonGroupTip= 'Change HiPS projection between spherical (180 deg) and Aitoff (360 deg)';
     return (
-        <RadioGroupInputFieldView {...{
-            options: projOptions, value, buttonGroup:true, title:buttonGroupTip,
-            sx: {pl:.5, button:{height:'1.5em'}},
-            onChange: (ev) => dispatchChangeCenterOfProjection({plotId:pv.plotId, fullSky:ev.target.value==='aitoff'})
-        }}/>
+        <DropDownToolbarButton {...{
+            sx:{button:{minWidth: '11rem', px:0}},
+            text,
+            tip:'Change HiPS projection between spherical (180 deg) and Aitoff (360 deg)',
```

Comment:
This tooltip is outdated, as it only refers to one function of the menu.

For the menu, perhaps "Change image orientation and projection".

Because the equatorial vs galactic choice is first in the dropdown name, it should also be first in the menu.  I would name it just "Orientation" and the mouseover for that can be "Choose the orientation of the HiPS all-sky image".

I'm avoiding "coordinate system" because I've realized that it may conflict in the users' minds with their choice of what coordinate system to _overlay_ on the image.

The "HiPS Projection" part of the menu would now be second, and I would just call it "Projection" in line with the comments about reducing the prominence of the technical term "HiPS".  The mouseover for this subsection header in the menu can then be "Choose the projection for the all-sky HiPS image".